tests/e2e/galleryitem.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { expect, test } from '@playwright/test'
+
+test('Organic user test of gallery item page', async ({ page }) => {
```

Comment:
I would create more test case with less checking like
- `User can share item`
- `User can see details & activity`
- `User can download item`

idk I might be wrong but that what I would test

Reply:
i plan to do both, wdyt? one set of tests verifies that elements are showing properly and another one checks functionality

Reply:
Idk sounds good, can't really help on how to acheive that on a larger scale as it's not really my field of choice